src/Assembly.js

```diff
@@ -203,24 +194,38 @@ class Assembly extends React.Component {
     other: null,
   }
 
-  // Current strip report
   @observable photos_uploaded = []
-
-  @observable language = "Español"
-
   @observable survey_anySymptoms = null
   @observable survey_tookMedication = null
 
-  @observable alerts = []
-  @observable provider = null
-
   @observable capturing = false
 
+  // ------ Coordinator ------
+
+  @observable coordinator_uuid = null
+
+  @observable coordinator_registration = new Account("Coordinator", {
+    name: "",
+    email: "",
+    password: "",
+  })
+
+  @observable coordinator_login_credentials = {
+    email: "",
+    password: "",
+  }
+
+  @observable coordinator_participants = new Records("Coordinator", "Participant", "participants")
+
+  // ------ Misc ------
+
+  @observable language = "Español"
+  @observable alerts = []
+  @observable currentPage = Login
+
   constructor(props) {
     super(props)
 
-    if(!this.authorized) { this.currentPage = Login }
```

Comment:
We will need this to keep not authorized users outside of our app

Reply:
Don't worry!
On line 183, we set the default page to `Login`.

You're right, though - we can be doing more to keep un-authenticated people out.
This line on its own just redirects the user when they first hit the app; it doesn't force people to log in.

I created #95 to follow up on this.